aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapter.java

```diff
@@ -732,16 +732,11 @@ private boolean dataExistsInSQLiteTable(
             @NonNull String columnName,
             @NonNull String columnValue) {
         // SELECT * FROM '{tableName}' WHERE {columnName} = '{columnValue}'
-        final String queryString = new StringBuilder()
-                .append(SqlKeyword.SELECT).append(SqlKeyword.DELIMITER)
-                .append("*").append(SqlKeyword.DELIMITER)
-                .append(SqlKeyword.FROM).append(SqlKeyword.DELIMITER)
-                .append(Quotes.wrapInSingle(tableName)).append(SqlKeyword.DELIMITER)
-                .append(SqlKeyword.WHERE).append(SqlKeyword.DELIMITER)
-                .append(columnName).append(SqlKeyword.DELIMITER)
-                .append(SqlKeyword.EQUAL).append(SqlKeyword.DELIMITER)
-                .append(Quotes.wrapInSingle(columnValue))
-                .toString();
+        final String queryString = "" +
+            SqlKeyword.SELECT + SqlKeyword.DELIMITER + "*" + SqlKeyword.DELIMITER +
```

Comment:
Outside of the bounds of this PR, but we don't need to actually transit data from SQLite here. You could change `"*"` to `"1"` given all we are doing is checking the count of returned rows in the cursor.

Reply:
Aha! TIL https://stackoverflow.com/a/7171072/695787

Let's do it in a separate PR